src/components/renderer/FramedDocumentRenderer.tsx

```diff
@@ -6,18 +6,18 @@ import { HostActions } from "../frame/host.actions";
 import { FrameActions, obfuscateField, updateHeight, updateTemplates } from "../frame/frame.actions";
 import { HostConnector } from "../frame/HostConnector";
 import { DomListener } from "../common/DomListener";
-import { noAttachmentRenderer } from "./NoAttachmentRenderer";
+import { renderableAttachmentRenderer } from "./RenderableAttachmentRenderer";
 import { OpenAttestationDocument, WrappedDocument } from "@govtechsg/open-attestation";
 
 const { trace } = getLogger("FramedDocumentRenderer");
 
 interface FramedDocumentRendererProps<D extends OpenAttestationDocument = OpenAttestationDocument> {
   templateRegistry: TemplateRegistry<D>;
-  attachmentToComponent?: (attachment: Attachment, document: OpenAttestationDocument) => React.FunctionComponent;
+  attachmentToComponent?: (attachment: Attachment, document: OpenAttestationDocument) => React.FunctionComponent | null;
 }
 export function FramedDocumentRenderer<D extends OpenAttestationDocument = OpenAttestationDocument>({
   templateRegistry,
-  attachmentToComponent = noAttachmentRenderer
+  attachmentToComponent = renderableAttachmentRenderer
```

Comment:
I removed it because it grThis will grow the bundle size even if you dont use the pdf. That's the reason it has been removed

Reply:
Reverted, but changed noAttachmentRender to hide tabs too.